test/test_vim.py

```diff
@@ -90,9 +90,15 @@ def test_vars(vim):
 
 
 def test_options(vim):
-    assert vim.windows[0].options['listchars'] == 'tab:> ,trail:-,nbsp:+'
-    vim.windows[0].options['listchars'] = 'tab:xy'
-    assert vim.windows[0].options['listchars'] == 'tab:xy'
+    assert vim.options['listchars'] == 'tab:> ,trail:-,nbsp:+'
```

Comment:
but this didn't work on master right? In the meanwhile we should use some other global option.

Reply:
But that way it should work with `'listchars'` being either global _or_ locally?

Reply:
Oh, it does not. Nevermind then. I'll use another option.

Reply:
Hmm are you sure? Was this test not changed recently because it did fail with nvim master?

Reply:
> Hmm are you sure? Was this test not changed recently because it did fail with nvim master?

yes https://github.com/neovim/pynvim/commit/5a329f221123efbb5a5515692260eb87469ebd39

Reply:
`vim.options['listchars']` doesn't work for me in manual testing on nvim master. (One can argue it _should_ work, but it doesn't).

Reply:
No, sorry, that was my bad. `vim.options['listchars']` wouldn't work with HEAD.